### ISSUE TITLE
dev3: backend delete events:  strange comportment!

### ISSUE THREAD

**Issue by jojo12** [jemproject/JEM-Project#1643]
I wanted to trash 3 events. It shows me, that they are trashed, but they are still inside the published, and I can't delete them.
Same, when I want to archive them, the info says, they are archived, but they are still inside the published.
Same, when I want to hide them.
I tested on 2 sites!

Please, test too: if this is really like this, this is a releaseblocker!

**Comment by Heklaterriol** [comment]
What do you mean by "inside the published"? visible in the frontend, in the event listing in the backend …

**Comment by Heklaterriol** [comment]
And where do you trash them, backend, frontend,
an by what method: opening the event an changeing the status, choosing the events in the listing and choosing the button ""unpublish", "archive", "trash"?

**Comment by Heklaterriol** [comment]
Works fine here

https://github.com/jemproject/JEM-Project/assets/28540581/1d16e709-5f54-4cd2-b07b-152f85b432da

Or are you talking about something else?

**Comment by jojo12** [comment]
I went back to your package and installed pkg_jem_v4.0.1-dev2.zip. There it is no problem.
The problem must be inside of the package, I made from the jem 4.0.1-dev branch.

**Comment by Heklaterriol** [comment]
I will install and check actual state of the branch.
Then compare pkg_jem_v4.0.1-dev2.zip with the actual files of the branch and see if I find the bug.

**Comment by Heklaterriol** [comment]
I can reproduce it with actual state of the branch.

**Comment by jojo12** [comment]
Thanks!

**Comment by Heklaterriol** [comment]
I checked the differences beween the two versions (pkg_jem_v4.0.1-dev2 and actuel state) and it seems that the fix #1590 for "Frontend Edit event: "Call to a member function format() on bool" if you use not english (date)" is causing this problem.

That means the bug is somewhere in here:
https://github.com/jemproject/JEM-Project/commit/2b5ceec2553fe472a9f6e95a9c7748d3f4721da9
https://github.com/jemproject/JEM-Project/commit/2b5ceec2553fe472a9f6e95a9c7748d3f4721da9

@mckillo: this is beyond my skills. Do you have time/capacities to check this?

**Comment by jojo12** [comment]
@mckillo: I agree, it's only, if we can't debug then to decide we prefere which bug ;-). But I am sure you will succeed. But do you have the time to do this?

**Comment by mckillo** [comment]
Guau! This is serious bug. I don't know. I have simulate the issue, an it's important to solve it. I hope to resolveit today, I'm going to find time in the day to debug.

**Comment by mckillo** [comment]
Fixed. Please, this last commit needs a tester.

**Comment by Heklaterriol** [comment]
Thanks @mckillo 
I am on holidays, so not very active. I will check it when I'm home again.

**Comment by mckillo** [comment]
Me too, it's time to relax and family.

**Comment by Heklaterriol** [comment]
even if it is holidays:
I cheked it and it works. Well done!

**Comment by jojo12** [comment]
Hi holidayers :-)
I am back and try to understand and test the history between 13 and 20 of August.
I checked the branch of mckillo's commit. It works, I tested on backend (and frontend).
(But in this branch, it seems, Marcos just integraded his solution with original bigger images for event details view. And with this I am not yet happy. Better to let it as it was - with flyer2 for overrides - this doesn't hurry!) The  actual issue here is important!!!
